NEWS.rst

```diff
@@ -1,6 +1,13 @@
 Version History
 ===============
 
+
+Version 0.6.0 (2020-12-01):
+* Added spatially bandlimited line source
+* Extended documentation of various functions
+* Various bug fixes
```

Comment:
Sure, I guess that's true for every single release, therefore it seems redundant to me.

Reply:
You are right, I will remove these generic statements.